pythonforandroid/recipes/Pillow/patches/fix-docstring.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/PIL/__init__.py b/src/PIL/__init__.py
```

Comment:
Minor, FWIW I would reference the upstream bug https://github.com/python-pillow/Pillow/issues/3231